blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -58,6 +58,8 @@ export class Activity extends Component {
 
             config.pipeline = pipeline;
             this.props.fetchRunsIfNeeded(config);
+
+            this.props.fetchCapabilitiesIfNeeded(this.props.pipeline._class);
```

Comment:
Why do you mean @tfennelly? ...but I agree that should be done in PipelinePage since it maybe needed for all children in the component tree.

...later more